docs/pages/access-controls/device-trust/enforcing-device-trust.mdx

```diff
@@ -199,8 +200,58 @@ version: v2
 ```
 
 Now the alice user can only access `env:production` apps using a trusted device.
-For example, to access an app called `myapp`, alice runs `tsh proxy app myapp -p
-8888` and opens http://localhost:8888 in her browser.
+
+## Desktop Access support
```

Comment:
We already have a header for "app access", so I'm not sure about what to do with reviewdog here. Any suggestions @ptgott?

Reply:
I think "Desktop access" is fine, since it makes sense as a task that someone would want to achieve with Teleport (without construing "Desktop Access" as a Teleport feature/product).